src/main/java/io/cdap/plugin/sfmc/connector/MarketingCloudConnector.java

```diff
@@ -78,6 +78,8 @@ public class MarketingCloudConnector implements DirectConnector {
   private Schema schema;
   private Iterator<? extends ETApiObject> iterator;
 
+  private static String timeOut = "300000";
```

Comment:
Use 'private static **final**', and rename to CXF_TIMEOUT_VALUE

Reply:
Renamed to 'CXF_TIMEOUT_VALUE' and Using private static final.